src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -45,6 +44,38 @@ See the
 [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
+## Resources and Process Tags
```

Comment:
Add a sentence at the beginning of this doc stating that, this is jaegerexporter which converts opentelemtry traces into jager model, following this spec - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/jaeger.md

Reply:
Also clarify this only supports Thrift over UDP.

Reply:
Added at the front paragraph at top